build/webcomponents/js/field-switcher/field-switcher.js

```diff
@@ -5,6 +5,16 @@
 		SPACE: 32,
 	};
 
+	const template = document.createElement('template');
+	const path = document.currentScript.src;
+	template.innerHTML = `<link href="${path.replace('/js/', '/css/').replace('.js', '.css').replace('-es5', '')}" rel="stylesheet" type="text/css"></link>
```

Comment:
`type="text/css"` << not needed

Reply:
unfortunately `document.currentScript` doesn't work with IE, so we have to encapsulate the css here. To ease the pain for newcomers we will also provide a PWA where someone will be able to tweak the colours or the scss and get an installable plugin that will create overrides to the current template and then self destruct. (or something like that...)

Reply:
Im simply referring to the `type` attribute which is not required in this case when using HTML5